examples/wmt15/config_large.yaml

```diff
@@ -0,0 +1,9 @@
+input_features:
+  - name: en
+    type: text
+    encoder: bert
+    pretrained_model_name_or_path: bert-base-uncased
+
+output_features:
+  - name: fr
+    type: text
```

Comment:
What about tokenization? we could either tokenize with space for French or with a workpiece tokenizer similar to the one used for the transformer encoder

Reply:
`french_tokenize` seems like a reasonable one to use.